src/main/java/com/uid2/admin/job/jobsync/PrivateSiteDataSyncJob.java

```diff
@@ -101,6 +103,14 @@ public void execute() throws Exception {
                 clock,
                 fileManager);
 
+        ClientSideKeypairStoreFactory keypairStoreFactory = new ClientSideKeypairStoreFactory(
```

Comment:
i would suggest to comment out all the codes you have added as we aren't generating keypair data for private operator yet for now.

Reply:
commented the rest of this stuff out in PrivateSiteDataSyncJob